research/conditional/train/cc_train.py

```diff
@@ -71,6 +71,9 @@ def main(
     VOCAB_SIZE = 30522 if args.model_type == "bert" else 50258
     DEVICE = torch.device("cuda" if torch.cuda.is_available() else "cpu")
 
+    if args.detect_anomaly:
+        torch.autograd.set_detect_anomaly(True)
```

Comment:
co to wgl jest? to wygląda na niezależną zmianę, co nie?

Reply:
po częśći niezależna, ale była używana do debugowania niestabilności grupowania